urdf2mjcf/model.py

```diff
@@ -80,6 +80,8 @@ class ConversionMetadata(BaseModel):
     freejoint: bool = True
     collision_params: CollisionParams = CollisionParams()
     joint_params: list[JointParam] | None = None
+    # Maps joint names to their metadata including actuator_type
+    joint_name_to_metadata: dict[str, dict[str, int | float | str]] | None = None
```

Comment:
it would be better for the values in the dictionary to be typed as well rather than a dict of primitives - see for example `CollisionParams`